README.md

```diff
@@ -106,11 +107,11 @@ Navigate to the `Message Body` tab and change some of the contents of the messag
 
 ### Viewing retries
 
-Return to the showcase UI to see that the retries have now been successfully handled.
+Return to the Showcase UI to see that the retries have now been successfully handled.
```

Comment:
In the document, there are several instances where 'showcase' has the letter 's' in lowercase. Should we update all of them to uppercase?